micromanager_gui/_gui_objects/_camera_widget.py

```diff
@@ -0,0 +1,124 @@
+from qtpy import QtWidgets as QtW
+from qtpy.QtCore import Qt
+
+
+class MMCameraWidget(QtW.QWidget):
+    """
+    Contains the following objects:
+
+    camera_groupBox: QtW.QGroupBox
+    bin_comboBox: QtW.QComboBox
+    bit_comboBox: QtW.QComboBox
+    px_size_doubleSpinBox: QtW.QDoubleSpinBox
+    cam_roi_comboBox: QtW.QComboBox
+    crop_Button: QtW.QPushButton
+    """
+
+    def __init__(self):
+        super().__init__()
+        self.setup_gui()
+
+    def setup_gui(self):
+
+        self.main_layout = QtW.QGridLayout()
+        # camera groupbox in widget
+        self.camera_groupBox = QtW.QGroupBox()
+        self.camera_groupBox.setTitle("Camera")
+        self.camera_groupBox.setMaximumHeight(135)
+        self.main_layout.addWidget(self.camera_groupBox, 0, 0)
+        self.setLayout(self.main_layout)
+
+        # define camera_groupBox layout
+        self.camera_groupBox_layout = QtW.QGridLayout()
+        # self.camera_groupBox_layout.setSpacing(0)
+        self.camera_groupBox_layout.setHorizontalSpacing(3)
+        self.camera_groupBox_layout.setVerticalSpacing(0)
+        self.camera_groupBox_layout.setContentsMargins(5, 5, 5, 0)
+        # add to camera_groupBox layout:
+        # bin widget and layout
+        self.bin_wdg = QtW.QWidget()
+        self.bin_layout = QtW.QGridLayout()
+        self.bin_layout.setSpacing(0)
+        self.bin_layout.setContentsMargins(0, 0, 3, 0)
+        # label bin in layout
+        self.bin_label = QtW.QLabel(text="Binning:  ")
+        self.bin_label.setMaximumWidth(65)
+        self.bin_layout.addWidget(self.bin_label, 0, 0)
+        # combobox bin in layout
+        self.bin_comboBox = QtW.QComboBox()
+        self.bin_layout.addWidget(self.bin_comboBox, 0, 1)
+        # set bin_wdg layout
+        self.bin_wdg.setLayout(self.bin_layout)
+        # bin widget in groupbox
+        self.camera_groupBox_layout.addWidget(self.bin_wdg, 0, 0)
+
+        # bit widget and layout
+        self.bit_wdg = QtW.QWidget()
+        self.bit_layout = QtW.QGridLayout()
+        self.bin_layout.setSpacing(0)
+        self.bit_layout.setContentsMargins(0, 0, 3, 0)
+        # label bit in groupbox r1 c0
+        self.bit_label = QtW.QLabel(text="Bit Depth:")
```

Comment:
For me at least this is not rendering correctly:
![image](https://user-images.githubusercontent.com/10111092/155829277-59b577ce-cd1a-4e71-8b1d-25023d2b76af.png)

Reply:
turns out this was bad before for me. Not a blocker

Reply:
As in #107 I think we can remove the widgets for binning and bit depth.